css/style.css

```diff
@@ -288,5 +290,209 @@ footer p {
   font-weight: 600;
 }
 
+@media screen and (max-width: 768px) {
+
+   .container {
+      font-size: 0.5 rem;
+      width: 100%;
+      text-align: center;
+      margin:auto;
+   }
+
+   .logo {
+      font-size: 0.5 rem;   }
+
+   header {
+      background: #033048;
+      padding: 5px 20px;
+      overflow: hidden;
+      width:100%;
+      text-align: center;
+    }
+
+   header nav ul li {
+      display: inline-block;
+      margin: 0 0 0 20px;
+      font-size: 20px;
+      float:left;
+    }
+
+    #hero {
+      width: 100%;
+      height: 50%;
+      background: url(../images/header_bg.jpg) no-repeat top center #131e27;
+      padding-top: 15px;
+      padding-bottom: 5px;
+    }
+
+#hero h2 {
+  font-size: 24px;
+  text-align: center;
+  color: #fff;
+  margin: 30px 0 30px;
+  font-weight: 600;
+}
+.yellow{
+  border-radius: 5px;
+  font-weight: 600;
+  padding: 12px 0;
+  width: 240px;
+  font-size: 28px;
+  text-align: center;
+  display: block;
+  margin: 0 auto;
+  margin-bottom: 200px;
+}
+
+.blue {
+  background: #033048;
+  color: #fff;
+  font-size: 20px;
+  border-radius: 5px;
+  font-weight: 600;
+  padding: 12px 0;
+  width: 240px;
+  font-size: 20px;
+  text-align: center;
+  display: block;
+  margin: 0 auto;
+}
+
+#benefits {
+  padding: 15px 10px 25px;
+}
+h3 {
+  color: #033048;
+  font-size: 30px;
+  font-weight: 600;
+  margin-bottom: 20px;
+}
+#benefits p, #benefits ul {
+  width: 100%;
```

Comment:
change this to `max-width: 100%`; since yo have a left and right margin on the `#benefits ul`, the 100% width + left and right margin on the UL is actually exceeding 100% width - thus its causing horizontal scrolling on small screens